### PR TITLE
Add 'Guide to Pharmacology' as drug prefix

### PR DESCRIPTION
**Pull request by dustine32**
This is the fix for that "failed drug test" in #135. As such I've removed the hard-coded release version (2021-02-01) of the `reacto.owl` used by the tests. I actually switched the conversion code to start pulling these drug xref mappings directly from the Reactome BioPAX rather than [checking the external](https://github.com/geneontology/pathways2GO/blob/9ac81db73cfff952739b6701203132a05837fb80/exchange/src/main/java/org/geneontology/gocam/exchange/BioPaxtoGO.java#L974) `reacto.owl`. Since REACTO is generated from the Reactome BioPAX anyway it seemed like an unnecessary lag step (generate `reacto.owl`, wait for GO release).

Basically, this creates a drug prefix list for "IUPHAR" and adds "Guide to Pharmacology". This is needed because drug mappings are now coming through the Reactome BioPAX like:
```
  <bp:UnificationXref rdf:ID="UnificationXref97504">
    <bp:db rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Guide to Pharmacology</bp:db>
    <bp:id rdf:datatype="http://www.w3.org/2001/XMLSchema#string">7168</bp:id>
  </bp:UnificationXref>
```

### CODE REVIEW

**Comment by dustine32** [comment]
Merging this now as it's been successfully tested in models pushed to noctua-dev and prod.

**Comment by deustp01** [comment]
@dustine32 Naive question: is this intended to make the filtering-out of reactions involving drugs in the build of GO-CAMs from Reactome more reliable, or something else? (If something else, no explanation needed.)

**Comment by dustine32** [comment]
@deustp01 Oh this is nothing new, just merging the code that we used for the new models in #136 into the main code branch. I'm mainly doing this to cleanly add the code fix for the drug sets issue #182.
